### PR TITLE
Doc db

### PR DESCRIPTION
**Pull request by sebastus**
I was able to squash commits this time.

### CODE REVIEW

**Comment by bingosummer** [comment]
@sebastus 
Could you resolve the conflicts in your PR?

**Comment by bingosummer** [comment]
The commits should be merged into the `master` branch instead of the `RedisCache` branch.

File: lib/common/reply.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var Reply = function(statusCode, altStatusCode, altCode) {
+    var reply = {};
+    switch(statusCode) {
+        case 200:
+            reply = {
+                statusCode: altStatusCode || 200,
+                code: altCode || 'Accepted',
```
Comment (bingosummer) [inline lib/common/reply.js]: The code for 200 should be `OK`.

**Comment by sebastus** [comment]
closing - I'll PR to master.

**Pull request closed without merging**